Fix io-ts errors due to new library version

## description
New io-ts versions (> 1.3.1) make toctoc web project fail to build with the following error:
```
node_modules/io-ts/lib/index.d.ts(8,29): error TS2304: Cannot find name 'unknown'
```

## how to reproduce
Just build the project

--- a/web/package.json
+++ b/web/package.json
@@ -53,7 +53,7 @@
   "dependencies": {
     "axios": "^0.17.1",
     "cookies-js": "^1.2.3",
-    "io-ts": "1.3.1"
+    "io-ts": "^1.5.1"
   },
   "devDependencies": {
     "@types/jest": "^21.1.2",
@@ -63,6 +63,6 @@
     "ts-jest": "^21.1.2",
     "tslint": "^5.7.0",
     "tslint-config-standard": "^6.0.1",
-    "typescript": "^2.8.1"
+    "typescript": "^3.2.2"
   }
 }

